addon/components/new-osf-navbar/template.hbs

```diff
@@ -35,9 +35,7 @@
             </div>
             <div class="navbar-collapse collapse navbar-right" id="secondary-navigation">
                 <ul class="nav navbar-nav">
-                    {{!SECONDARY NAVIGATION - displays corresponding links for selected service}}
```

Comment:
This comment can stay, I just meant the "observable value" line wasn't necessary.